libraries/Examples/examples/Example8_ResetsAndWatchdog/Example8_ResetsAndWatchdog.ino

```diff
@@ -0,0 +1,196 @@
+/* Author: Stephen Fordyce (adapted from WDT example code by Adam Garbo - https://forum.sparkfun.com/viewtopic.php?f=169&t=52431&p=213296&hilit=watchdog#p213296)
```

Comment:
This example includes a lot of great functionality!
It is fairly overwhelming as the *only* WDT example - it would be great to have a WDT library with several bite-sized examples

Reply:
Hi Owen, sounds like I'm outvoted on the RTC library, no problem.  Also noted the preference for doing separate smaller PRs for different features (makes sense).

Github implies you've requested a change on line 1 (or line 196?) of Example8_ResetsAndWatchdog.ino but I can't see what it is - can you explain what's going on?  (still a bit of a Github numpty here)

I agree a WDT library and some smaller examples would be better, but I've expended my energy budget on this for the moment (no problem from me if someone wants to pick it up and run with it).  I deliberately structured the example as it is, so it's also an all-in-one-place reference for what HAL calls are involved and the lower level stuff.  Saving advanced user from having to pick through various files, and giving less advanced users a good chance at copy/pasting functionality.  Ie. it should stand alone, and can be replaced/supplemented by library examples later.  I figured better to have this in the core as an example than posted on a forum. 

Do you want me to cancel this multi-faceted pull request and do another one just for the Reset/Watchdog example?

Cheers,
Stephen

Reply:
Yes, that would be great. Thank you. I agree, we can build smaller examples over time but for now lets get it in here!